server/routes/serviceProviderReferrals/scheduleAppointmentPresenter.test.ts

```diff
@@ -76,7 +76,7 @@ describe(ScheduleAppointmentPresenter, () => {
           expect(presenter.appointmentSummary).toEqual([
             { key: 'Caseworker', lines: ['firstName lastName'] },
             { key: 'Date', lines: ['2 January 2021'] },
-            { key: 'Time', lines: ['12:00pm to 1:00pm'] },
+            { key: 'Time', lines: ['midday to 1:00pm'] },
```

Comment:
This should probably be capitalised? Ditto elsewhere when it's at the start of a sentence.

Reply:
Thank you. I've added a new commit for this.